scripts/partition_manager/partition_manager.rst

```diff
@@ -261,16 +259,16 @@ It includes :file:`autoconf.h` (which is generated by Kconfig) and uses a Kconfi
 
 Build system
 ************
-The Partition Manager uses Zephyr's multi-image build system to collect configurations for all sub-images.
-If one or more sub-images are included in a build, their names are appended to a global list.
+The Partition Manager uses Zephyr's multi-image build system to collect configurations for all child images.
```

Comment:
```suggestion
The Partition Manager uses the |NCS| multi-image build system to collect configurations for all child images.
```

Reply:
I think it's safe to make this change now that we're sure this is NCS only for the foreseeable future, while you're making changes to this file. But no worries if you consider this to be out of scope of this pull request.

Reply:
I'm going to merge now and let you decide what to do with this suggestion.

Reply:
seems we are addressing this in the PR that migrates us to the new multi-image implementation.

https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1508/files#diff-9c00373859ec51bef48fc873f0a99dd2R264